docker-compose.yml

```diff
@@ -34,12 +34,12 @@ services:
         ipv4_address: 173.23.0.100
 
   mongodb:
-    image: 'mongo:3.4-xenial'
+    image: 'mongo:7'
     networks:
       - etl-network
 
   mysql:
-    image: mysql
+    image: mysql:8
```

Comment:
Pegged mysql and postgres to specific versions.  Otherwise, these become the latest versions, and existing tests that previously succeeded begin to ail.